components/layout/menu.hbs

```diff
@@ -1,316 +1,293 @@
 <nav
-  aria-label="Main Navigation"
-  id="widescreen-menu"
-  class="widescreen-menu show-for-large"
-  role="menu"
+  id="top-nav"
+  class="top-nav group no-utility-banner has-snap-panel section--bg-prim-blue"
```

Comment:
For this, let's keep it limited to only the things we need, so I think you could get ride of all the `top-nav group no-utility-banner has-snap-panel` classes and the ID.